src/tracing/json/json_tracing.cpp

```diff
@@ -84,7 +97,17 @@ ::Json::Value GetPayload(PayloadDecoderType & decoder)
             value[formatter.Reset().AddFormat("EVNT(%u/%u)", entry.GetClusterId(), entry.GetEventId()).c_str()] = "<NOT_DECODED>";
             continue;
         default:
-            value[entry.GetName()] = entry.GetValueText();
+            formatter.Reset().Add(entry.GetName()); // name gets destroyed by decoding
```

Comment:
Again, please document.  And what does the "destroyed" comment mean?

Reply:
This does not apply here. Comment removed.